documentation/Tests.md

```diff
@@ -0,0 +1,11 @@
+# List of useful tests to be added
+
+- [ ] Upload file:
+  - [ ] Succesful added
+  - [ ] Connection error
+  - [ ] Permission error
+  - [ ] Overwrite error
```

Comment:
Good idea to add these tests.
As it's currently implemented, it shouldn't overwrite anything, since the filename is appended with the Database ID on upload. Not sure that this is the best way, but it seemed like a reasonable, simple way of ensuring filename uniqueness.